src/SprykerEco/Zed/Optivo/Communication/Plugin/OptivoRequestPluginInterface.php

```diff
@@ -9,7 +13,7 @@ interface OptivoRequestPluginInterface
     /**
      * @param \Spryker\Shared\Kernel\Transfer\AbstractTransfer $transfer
      *
-     * @return mixed
+     * @return void
      */
-    public function send(AbstractTransfer $transfer);
+    public function send(AbstractTransfer $transfer): void;
```

Comment:
It would be nice to avoid using AbstractTransfer and use more specific transfers. It could be problem for phpstan.

Reply:
I'have deleted it